src/Algebra/Morphism/TwoMagmas.agda

```diff
@@ -0,0 +1,52 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties of Magma homomorphisms
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+
+open import Algebra using (Magma)
+import Algebra.Morphism.Definitions as Definitions
+open import Data.Product using (_,_)
+open import Function using (id; _∘_)
+open import Function.Definitions using (Inverseᵇ)
+open import Relation.Binary using (_Preserves_⟶_)
+import Relation.Binary.Reasoning.Setoid as EqR
+open import Relation.Unary using (Pred)
+
+
+module Algebra.Morphism.TwoMagmas {α α= β β=} (M : Magma α α=) (M' : Magma β β=)
+  where
+open Magma M using (_≈_; _∙_; setoid) renaming (Carrier to A; refl to ≈refl)
+open Magma M' using () renaming (Carrier to B; _≈_ to _≈'_; sym to ≈'sym;
+                                 _∙_ to _∙'_; ∙-cong to ∙'cong; setoid to setoid')
+
+open Definitions A B _≈'_ using (Homomorphic₂)
+open Definitions B A _≈_  using () renaming (Homomorphic₂ to Homomorphic₂')
+open EqR setoid
+
+IsHomo :  Pred (A → B) _
+IsHomo f =  Homomorphic₂ f _∙_ _∙'_
+
+IsHomo-rev :  Pred (B → A) _
+IsHomo-rev f =  Homomorphic₂' f _∙'_ _∙_
```

Comment:
I don't think these aliases add anything. Please use the full expression.

Reply:
I set the full expressions.